app/controllers/organisations_controller.rb

```diff
@@ -11,17 +11,22 @@ def index
     redirect_to organisation_applicants_path(@organisations.first) if @organisations.to_a.length == 1
   end
 
-  def show; end
-  def edit; end
+  def show
+    render partial: "organisation", locals: { organisation: @organisation }
+  end
+
+  def edit
+    render partial: "organisation_form", locals: { organisation: @organisation }
+  end
 
   def update
     @organisation.assign_attributes(**organisation_params)
     authorize @organisation
     if update_organisation.success?
-      render :show
+      render partial: "organisation", locals: { organisation: @organisation }
     else
       flash.now[:error] = update_organisation.errors&.join(",")
-      render :edit, status: :unprocessable_entity
+      render partial: "organisation_form", status: :unprocessable_entity, locals: { organisation: @organisation }
     end
```

Comment:
personnellement je trouve ça toujours confusant et pas vraiment aligné avec the "rails" way.
Je préférerais garder une vue `edit` et `show`, 
et apellé `render :edit` ou `render :show` dans l'action `update`.
Je ne pense pas que ce soit au controller de savoir quelle partial appeler avec quelles variables.

Reply:
Ok, vous avez raison. J'ai rollback ce changement.